doc/source/cluster/aws-tips.rst

```diff
@@ -71,3 +71,202 @@ You should see something like
+                config: "cloudwatch/example-cloudwatch-agent-config.json"
+            dashboard:
+                # CloudWatch Dashboard name
+                name: "example-dashboard-name"
+                # Path to the CloudWatch Dashboard config file
+                config: "cloudwatch/example-cloudwatch-dashboard-config.json"
+
+    auth:
+        ssh_user: ubuntu
+
+    available_node_types:
+        ray.head.default:
+            node_config:
+            InstanceType: c5a.large
+            ImageId: ami-0d88d9cbe28fac870  # Unified CloudWatch agent pre-installed AMI, us-west-2
+            resources: {}
+        ray.worker.default:
+            node_config:
+                InstanceType: c5a.large
+                ImageId: ami-0d88d9cbe28fac870  # Unified CloudWatch agent pre-installed AMI, us-west-2
+                IamInstanceProfile:
+                    Name: ray-autoscaler-cloudwatch-v1
+            resources: {}
+            min_workers: 0
+
+2. Download example CloudWatch agent and dashboard configuration files:
+
+   First, create a ``cloudwatch`` directory in the same directory as ``cloudwatch-basic.yaml``.
+   Then, download the `CloudWatch agent config <https://github.com/ray-project/ray/blob/master/python/ray/autoscaler/aws/cloudwatch/example-cloudwatch-agent-config.json>`_ and `CloudWatch dashboard config <https://github.com/ray-project/ray/blob/master/python/ray/autoscaler/aws/cloudwatch/example-cloudwatch-dashboard-config.json>`_ to the cloudwatch directory.
+
+3. Run ``ray up cloudwatch-basic.yaml`` to start your Ray Cluster.
+
+.. note::
+    This will launch your Ray cluster in ``us-west-2`` by default. When launching a cluster for a different region, you'll need to change your YAML cluster launcher config file's ``region`` AND ``ImageId``.
+    See the ``Unified CloudWatch Agent Images table`` above for available AMIs by region.
+
+4. Check out your Ray cluster's logs, metrics, and dashboard in the `CloudWatch Console <https://console.aws.amazon.com/cloudwatch/>`_!
+
+Advanced Setup:
+~~~~~~~~~~~~~~~
+
+Refer to `example-cloudwatch.yaml <https://github.com/ray-project/ray/blob/master/python/ray/autoscaler/aws/example-cloudwatch.yaml>`_ for a complete example.
+
+1. Choose an AMI with the Unified CloudWatch Agent pre-installed.
+
+   Ensure that you're launching your Ray EC2 cluster in the same region as the AMI,
+   then specify the ``ImageId`` to use with your cluster's head and worker nodes in your YAML cluster config file:
+
+.. code-block:: yaml
+
+    available_node_types:
+        ray.head.default:
+            node_config:
+            InstanceType: c5a.large
+            ImageId: ami-0d88d9cbe28fac870 (us-west-2)
+        ray.worker.default:
+            node_config:
+            InstanceType: c5a.large
+            ImageId: ami-0d88d9cbe28fac870 (us-west-2)
+
+   The AMIs are based on version 48 of the Ubuntu 18.04 AWS Deep Learning AMI, and ship with Unified CloudWatch Agent v1.247348.0b251302.
```

Comment:
This is already noted in the above table and doesn't need to be restated here (the AMI version number is the only thing currently omitted from that table, but I also don't think it is critical to include initially).